src/torchjd/autojac/mtl_backward.py

```diff
@@ -70,6 +152,8 @@ def mtl_backward(
         functions. The arguments of ``mtl_backward`` should thus not come from a compiled model.
         Check https://github.com/pytorch/pytorch/issues/138422 for the status of this issue.
     """
+    if shared_params is None or tasks_params is None:
+        shared_params, tasks_params = _get_tasks_shared_params(losses)
```

Comment:
One could be non-`None`, in which case we want to keep its current value, so we need to `if` statement.